modules/gltf_blend/register_types.cpp

```diff
@@ -0,0 +1,74 @@
+/* Copyright (c) 2007-2021 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2021 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "register_types.h"
+
+#include "core/config/project_settings.h"
+#include "editor/editor_node.h"
+#include "editor_scene_importer_blend.h"
+
+#ifndef _3D_DISABLED
+#ifdef TOOLS_ENABLED
+static void _editor_init() {
+	_GLOBAL_DEF("filesystem/import/blend/enabled", true, true);
+	bool blender_enabled =
+			ProjectSettings::get_singleton()->get("filesystem/import/blend/enabled");
+	if (!blender_enabled) {
+		return;
+	}
+	_EDITOR_DEF("filesystem/blend/blender_path", "blender", true);
+	EditorSettings::get_singleton()->add_property_hint(
+			PropertyInfo(Variant::STRING, "filesystem/blend/blender_path",
+					PROPERTY_HINT_GLOBAL_FILE));
+	String blender_path =
+			EditorSettings::get_singleton()->get("filesystem/blend/blender_path");
+	if (blender_path.is_empty()) {
+		return;
+	}
+	Ref<EditorSceneFormatImporterBlend> import_gltf;
+	import_gltf.instantiate();
+	ResourceImporterScene::get_singleton()->add_importer(import_gltf);
+}
+#endif
+#endif
+
+void register_gltf_blend_types() {
+#ifndef _3D_DISABLED
+#ifdef TOOLS_ENABLED
+	ClassDB::APIType prev_api = ClassDB::get_current_api();
+	ClassDB::set_current_api(ClassDB::API_EDITOR);
+	ClassDB::set_current_api(prev_api);
+	GDREGISTER_CLASS(EditorSceneFormatImporterBlend);
+	EditorNode::add_init_callback(_editor_init);
```

Comment:
I think this is supposed to be like this:

```suggestion
	ClassDB::APIType prev_api = ClassDB::get_current_api();
	ClassDB::set_current_api(ClassDB::API_EDITOR);
	GDREGISTER_CLASS(EditorSceneFormatImporterBlend);
	ClassDB::set_current_api(prev_api);
	EditorNode::add_init_callback(_editor_init);
```

Reply:
Was only the C# CI failing? I thought I added some checks for this to the ClassDB tests.